packs/data/spell-effects.db/spell-effect-blessing-of-defiance.json

```diff
@@ -31,15 +31,84 @@
                         "value": "will"
                     }
                 ],
-                "flag": "blessingOfDefiance",
+                "flag": "blessingOfDefianceSave",
                 "key": "ChoiceSet",
                 "prompt": "PF2E.SpecificRule.BlessingOfDefiance.Prompt"
             },
             {
                 "key": "FlatModifier",
-                "selector": "{item|flags.pf2e.rulesSelections.blessingOfDefiance}",
+                "selector": "{item|flags.pf2e.rulesSelections.blessingOfDefianceSave}",
                 "type": "status",
                 "value": 4
+            },
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "option": "blessing-of-defiance-save:{item|flags.pf2e.rulesSelections.blessingOfDefianceSave}"
+            },
+            {
+                "key": "Resistance",
+                "predicate": [
+                    {
+                        "and": [
+                            "blessing-of-defiance-save:fortitude",
+                            "blessing-of-defiance-actions:two"
+                        ]
+                    }
+                ],
+                "type": "poison",
+                "value": 15
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    {
+                        "and": [
+                            "blessing-of-defiance-save:reflex",
+                            "blessing-of-defiance-actions:two"
+                        ]
+                    }
+                ],
+                "selector": "speed",
+                "type": "status",
+                "value": 15
+            },
+            {
+                "key": "Resistance",
+                "predicate": [
+                    {
+                        "and": [
+                            "blessing-of-defiance-save:will",
+                            "blessing-of-defiance-actions:two"
+                        ]
+                    }
+                ],
+                "type": "mental",
+                "value": 15
+            },
+            {
+                "choices": [
+                    {
+                        "label": "One Action",
+                        "value": "one"
+                    },
+                    {
+                        "label": "Two Actions",
+                        "value": "two"
+                    },
+                    {
+                        "label": "Three Actions",
+                        "value": "three"
+                    }
+                ],
+                "flag": "blessingOfDefianceActions",
```

Comment:
If you also add:
`"rollOption": "blessing-of-defiance-actions"`
You shouldn't need the RollOption RE.